rules/S6915/java/metadata.json

```
@@ -0,0 +1,25 @@
+{
+  "title": "\"String.indexOf\" should be used with correct ranges",
+  "type": "CODE_SMELL",
```

I know this field is here for legacy reason but I think we should stay consistent and set the type to "BUG"
```suggestion
  "type": "BUG",
```